guides/v2.2/ui_comp_guide/concepts/magento-bindings.md

```diff
@@ -0,0 +1,78 @@
+|style          |`<div data-bind="style: {color: redColor}"> </div>`                            | `<div ko-style="color: redColor"> </div>`                             |
+|html           |`<div data-bind="html: '<span/>'"> </div>`                                     | `<div html="'<span/>'"> </div>`                                       |
+|click          |`<div data-bind="click: onClick"> </div>`                                      | `<div click="onClick"> </div>`                                        |
+|event          |`<div data-bind="event: {mouseover: showEl}"> </div>`                          | `<div event="mouseover: showEl"> </div>`                              |
+|template       |`<div data-bind="template: templateUrl"> </div>`                               | `<div template="templateUrl"> </div>`                                 |
+|submit         |`<form data-bind="submit: onSubmit"> </form>`                                  | `<form submit="onSubmit"> </form>`                                    |
+|options        |`<select data-bind="options: optionsList"> </select>`                          | `<select options="optionsList"> </select>`                            |
+|selectedOptions|`<select data-bind="options: optionsList, selectedOptions: value"> </select>`  | `<select options="optionsList" selectedOptions="value"> </select>`    |
+|optionsText    |`<select data-bind="options: optionsList, optionsText: 'label'"> </select>`    | `<select options="optionsList" optionsText="'label'"> </select>`      |
+|optionsText    |`<select data-bind="options: optionsList, optionsValue: 'value'"> </select>`   | `<select options="optionsList" optionsValue="'value'"> </select>`     |
+|enable         |`<input data-bind="enable: isEnabled"/>`                                       | `<input enable="isEnabled"/>`                                         |
+|disable        |`<input data-bind="disable: !isEnabled"/>`                                     | `<input ko-disabled="!isEnabled"/>`                                   |
+|hasFocus       |`<input data-bind="hasFocus: onFocus"/>`                                       | `<input ko-focused="onFocus"/>`                                       |
+|textInput      |`<input data-bind="textInput: value"/>`                                        | `<input textInput="value"/>`                                          |
+|value          |`<input data-bind="value: value"/>`                                            | `<input ko-value="value"/>`                                           |
+|checked        |`<input type="chackbox" data-bind="checked: isChecked"/>`                      | `<input type="chackbox" ko-checked="isChecked"/>`                     |
+|checkedValue   |`<input type="chackbox" data-bind="checkedValue: $data, checked: isChecked"/>` | `<input type="chackbox" checkedValue="$data" checked="isChecked"/>`   |
```

Comment:
Append the following on the next line of this table to get a better table format:
```
{:style="table-layout:auto"}
```

Reply:
ah, awesome. will do.